File: cpp/Nilai.cpp
#include <iostream>
using namespace std;

class Nilai {
    private:
        float nilai;

    public:
        Nilai() {
            this->nilai=0;
        }

        Nilai(float val) {
            this->nilai=val;
        }

        void input(string penilai) {
            cout << "Masukkan nilai " << penilai << ": "; cin >> this->nilai;

        }

        void setNilai(float val) { this->nilai=val; }

        float getNilai() { return this->nilai; }
};

File: py/Nilai.py
class Nilai :
    def __init__(self) :
        self.__nilai=0.0
    
    def input(self,penilai) :
        self.__nilai=input("Masukkan nilai " + penilai + ": ")
    
    def setNilai(self,nilai) : self.__nilai=nilai

    def getNilai(self) : return self.__nilai